lib/rust/parser/generate-java/java/org/enso/syntax2/FinalizationManager.java

```diff
@@ -0,0 +1,54 @@
+package org.enso.syntax2;
+
+import java.lang.ref.*;
+import java.util.Collections;
+import java.util.IdentityHashMap;
+import java.util.Set;
+
+public final class FinalizationManager {
```

Comment:
I assume this is not a `public` API - please remove the `public` keywords in this file.